src/idpi/operators/pot_vortic.py

```diff
@@ -81,4 +81,6 @@ def fpotvortic(
         dt_dlam * curl1 + dt_dphi * (curl2 + cor2) - dt_dzeta * (curl3 + cor3)
     ) / rho_tot
 
+    out.attrs = theta.attrs
```

Comment:
how was this not relevant before?

Reply:
The handling of the attributes is not consistent throughout the library at the moment. The pot vortic output needs to be interpolated at theta values and integrated between pressure levels for the ninjo product so the vertical coordinate attributes are required.

Reply:
👍